generator/internal/client_generator.cc

```diff
@@ -354,7 +358,11 @@ Status ClientGenerator::GenerateCc() {
 
   CcPrint(  // clang-format off
     "\n"
-    "$client_class_name$::$client_class_name$(std::shared_ptr<$connection_class_name$> connection, Options opts)"
+    "$client_class_name$::$client_class_name$("
```

Comment:
nit: looks like this section could benefit from raw strings too.

Reply:
Done.